design/vela-core/workflow_policy.md

```diff
@@ -142,7 +142,7 @@ If the status of workflow step is `waiting` or `failed`, the workflow will be re
 int(0.05 * 2^(n-1))
 ```
 
-Based on the above formula, we will take `1s` min time.
+Based on the above formula, we will take `1s` as the min time and `60s` as the max time. You can change the max time by setting `MaxWorkflowWaitBackoffTime`.
```

Comment:
Now that we allow the workflow to terminate, do we need to incrementally increase the interval?

Reply:
Maybe we will add the following fields:
```
steps:
  - name: test
     properties:
        xxx: zzz
     retry: 
        interval: 2s
        max: 3
        timeout: 30s
```

Reply:
Should this `interval` influent our `backoff` mechanism?
IMO, the workflow should like:
```yaml
steps:
  - name: test
     properties:
        xxx: zzz
     retries: 5
     timout: 30s 
```
Anyway, we can leave this discussion to the next PR, this PR is mainly for `if`.